x-pack/plugins/observability/public/components/app/section/apm/index.tsx

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  Axis,
+  BarSeries,
+  Chart,
+  DARK_THEME,
+  LIGHT_THEME,
+  niceTimeFormatter,
+  Position,
+  ScaleType,
+  Settings,
+} from '@elastic/charts';
+import { EuiFlexGroup, EuiFlexItem, EuiStat } from '@elastic/eui';
+import numeral from '@elastic/numeral';
+import { i18n } from '@kbn/i18n';
+import moment from 'moment';
+import React, { useContext } from 'react';
+import { useHistory } from 'react-router-dom';
+import { ThemeContext } from 'styled-components';
+import { SectionContainer } from '../';
+import { getDataHandler } from '../../../../data_handler';
+import { FETCH_STATUS, useFetcher } from '../../../../hooks/use_fetcher';
+import { ChartContainer } from '../../chart_container';
+import { onBrushEnd } from '../helper';
+import { StyledStat } from '../../styled_stat';
+
+interface Props {
+  startTime?: string;
+  endTime?: string;
+  bucketSize?: string;
+}
+
+function formatTransactionValue(value?: number) {
+  return numeral(value).format('0.00a');
+}
+
+export const APMSection = ({ startTime, endTime, bucketSize }: Props) => {
+  const theme = useContext(ThemeContext);
+  const history = useHistory();
+
+  const { data, status } = useFetcher(() => {
+    if (startTime && endTime && bucketSize) {
+      return getDataHandler('apm')?.fetchData({ startTime, endTime, bucketSize });
+    }
+  }, [startTime, endTime, bucketSize]);
+
+  const { title = 'APM', appLink, stats, series } = data || {};
+
+  const min = moment.utc(startTime).valueOf();
+  const max = moment.utc(endTime).valueOf();
+
+  const formatter = niceTimeFormatter([min, max]);
+
+  const isLoading = status === FETCH_STATUS.LOADING;
+
+  const transactionsColor = series?.transactions.color || theme.euiColorVis1;
+
+  return (
+    <SectionContainer
+      minHeight={296}
```

Comment:
This is an oddly specific size. Where does it come from? Should it perhaps be a constant with a comment?

Reply:
Removed